InnoTopicWebsite/src/styles.sass

```diff
@@ -1,169 +1,191 @@
+  text-transform: uppercase
+  color: gray
+  font-size: 0.75rem
+
+.powered-by mat-chip
+  img
+    opacity: 10%
+  opacity: 50% !important
+  margin: 3px !important
+  padding: 5px 8px !important
+  background-color: white !important
+  box-shadow: none !important
+
+.comment
+  font-size: 60%
+  font-size: .9rem
+  color: gray
+  opacity: .6
+
+.innotopicLogo
+  fill: white !important
+
+.powered-by
+   background-color: red
+
+//@media print
+.mat-chip.mat-standard-chip
+  -webkit-print-color-adjust: exact
+  //background-color: yellow !important
+  color: #999 !important
+  font-size: .85rem !important
+  @media screen and (max-width: $break-medium)
+    font-size: .8rem !important
+    padding: 6px 10px
+    min-height: 25px
+  @media screen and (max-width: $break-small)
+    font-size: .75rem !important
+
+.topic-logo-img
+  @media screen and (max-width: $break-medium)
+    width: 16px !important
+    height: 16px !important
+
+.other-profile-icon-container
+  width: 1.2em !important
+  height: 1.2em
+   padding: 5px
+   background-color: #eee
+   border-radius: 50%
+  flex: none
+  & + a
+    word-break: break-all
+    margin-left: 5px
+
+app-user-other-profile-icon
+  vertical-align: middle
+
+.otherProfileIcon
+  width: 1rem !important
+  height: auto !important
+  // TODO: theme - color
+
+app-other-profile-user-name p
+  display: flex
+  align-items: center
+  justify-content: flex-start
+
+.dark-theme
+  .section-title
+   background-color: var(--ion-color-dark)
+
+  .mat-toolbar.mat-primary
+   background-color: var(--ion-color-dark)
+
+.section-title
+  background-color: var(--ion-color-primary)
+  margin-top: 25px
+  margin-left: 0px
+  color: var(--ion-color-primary-contrast)
+  padding: 15px
+  text-transform: uppercase
+  @media screen and (max-width: $break-medium)
+    margin-top: 20px
+    margin-bottom: 10px
+    padding: 10px
+
+.container
+  max-width: $break-medium
+  margin: auto
+  padding: 0px 10px
+  @media screen and (max-width: $break-medium)
+    padding: 0px
+
+model-viewer
+  width: 100vw
+  height: 100vh
+  display: block
+  position: fixed
+  top: 0
+  left: 0
+  z-index: -9999
+
+body
+  background-color: #e0e0e0
+// Adding grid layout for languages section just crosschecking
+.languages-grid
+  display: grid
+  grid-template-columns: repeat(2, 1fr) // Creates two columns with equal width
+  gap: 2px
+
+.language
+  display: flex
+  align-items: center
+  margin: 10px 0
+  transition: transform 0.3s ease, box-shadow 0.3s ease, border-radius 0.3s ease
+
+  &:hover
+    transform: scale(1.05)
+    border-radius: 20px 0 20px 0
+    box-shadow: -5px 5px 15px rgba(0, 0, 0, 0.1), 5px -5px 15px rgba(255, 255, 255, 0.8)
+
+  .app-flags
+    display: inline-block
+    width: 55px
+
+  label
+    width: 110px
+    text-transform: uppercase
+    color: var(--ion-color-secondary)
+    &::after
+      content: ""
+
+  span
+    word-break: break-word
+    width: calc(100% - 55px - 110px)
+
+#languages
```

Comment:
This should prolly be moved to natural-languages components, and I think a random bold is not necessary?